lib/high_voltage.rb

```diff
@@ -3,6 +3,7 @@
 
 require 'high_voltage/configuration'
 require 'high_voltage/constraints/root_route'
+require 'high_voltage/page_collector'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.